db/install.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<XMLDB PATH="admin/tool/abconfig/db" VERSION="20190829" COMMENT="XMLDB file for Moodle admin/tool/abconfig"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="../../../../lib/xmldb/xmldb.xsd"
+>
+  <TABLES>
+    <TABLE NAME="tool_abconfig_experiments" COMMENT="Table to store experiments">
```

Comment:
hmm can't find the right wiki doc but 90% sure there is a convention to always use the singular for tables names ie 'experiment' without the s